wsi/drm/drmplane.cpp

```diff
@@ -92,7 +92,13 @@ bool DrmPlane::Initialize(uint32_t gpu_fd,
   for (uint32_t j = 0; j < total_size; j++) {
     uint32_t format = supported_formats_.at(j);
     if (IsSupportedMediaFormat(format)) {
-      prefered_video_format_ = format;
+#ifdef PREFER_ABGR
+      // fixme? force to use ABGR8888 to WA h264 1080p 60fps issue
+      // http://jira01.devtools.intel.com/browse/OAM-62484
+      prefered_video_format_ = DRM_FORMAT_ABGR8888;
```

Comment:
You should check if this format is actually supported by this plane, else we cannot do anything use format.

Reply:
forgot to add these checks ?

Reply:
@kalyankondapally  this is WA. If we do anything too further, we may need to do PNP test again

Reply:
This WA has pass regression test

Reply:
We might end of up with wierd commit failures, resulting in video composition with rest of content in this case. This could go for verse.

however in this case, Video is always in full screen is it ?